tests/improver-nbhood/02-basic.bats

```diff
@@ -46,3 +46,20 @@
   rm "$TEST_DIR/output.nc"
   rmdir "$TEST_DIR"
 }
+
+@test "nbhood 'square' --radius-in-km=20 input output" {
```

Comment:
I don't know. It works but I haven't seen any rules for this in our tests. It would be good to know. Also, if the square neighbourhood test was put elsewhere, then would 02_basic.bats have to be renamed? For example to 02_basic_circular.bats?